sonic_platform_base/chassis_base.py

```diff
@@ -54,9 +57,25 @@ def __init__(self):
         # List of ThermalBase-derived objects representing all thermals
         # available on the chassis
         self._thermal_list = []
+
         self._voltage_sensor_list = []
         self._current_sensor_list = []
 
+        # Initialize voltage and current sensors lists from data file if available
+        try:
+            (platform_path, _) = device_info.get_paths_to_platform_and_hwsku_dirs()
+            self.sensors_yaml_file = platform_path + "/sensors.yaml"
+
+            with open(self.sensors_yaml_file, 'r') as f:
+                sensors_data = yaml.safe_load(f)
+                if 'voltage_sensors' in sensors_data:
+                    self._voltage_sensor_list = sensor_fs.VoltageSensorFs.factory(sensors_data['voltage_sensors'])
```

Comment:
@bmridul where is the change where sensormond is actually using these list of sensors?

Reply:
Sensormond retrieves the sensor list from the chassis class 

https://github.com/sonic-net/sonic-platform-daemons/blob/d8977f3608e5c9263d9b9d9b33087890637ac436/sonic-sensormond/scripts/sensormond#L196

Sensor list in the chassis class is populated by default from the sensors.yaml file if present, OR provided by the platform in the chassis implementation.